src/views/Rules/ListRules/index.jsx

```diff
@@ -64,9 +65,15 @@ function ListRules(props) {
   const [scheduledChanges, fetchScheduledChanges] = useAction(
     getScheduledChanges
   );
+  const [deleteAction, delRule] = useAction(deleteRule);
   const isLoading = products.loading || channels.loading || rules.loading;
+  // todo: delete errors aren't being shown
```

Comment:
We should probably create a GitHub issue or something so we can keep track of the action items.